examples/platform/linux/CommissionerMain.cpp

```diff
@@ -76,7 +76,7 @@ using namespace chip::app::Clusters;
 using namespace ::chip::Messaging;
 using namespace ::chip::Controller;
 
-class MyServerStorageDelegate : public PersistentStorageDelegate
```

Comment:
Is this customer server storage really needed? It seems to be the same as the code in `src/include/platform/KvsPersistentStorageDelegate.h`

Reply:
I will add this to a separate cleanup issue https://github.com/project-chip/connectedhomeip/issues/22459